src/VsixV3/EditorsPackage/Microsoft.VisualStudio.Editors.swixproj

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\..\build\Targets\ProducesNoOutput.Settings.targets" />
+  <Import Project="..\..\..\build\Targets\VSL.Versions.targets" />
+
+  <PropertyGroup>
+    <OutputArchitecture>neutral</OutputArchitecture>
+    <OutputLocalized>false</OutputLocalized>
+    <OutputType>manifest</OutputType>
+    <IsPackage>true</IsPackage>
+    <OutputName>Microsoft.VisualStudio.Editors</OutputName>
+    <UseVisualStudioVersion>true</UseVisualStudioVersion>
+  </PropertyGroup>
+
+  <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
+
+  <PropertyGroup>
+    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(AssemblyVersion);OutputPath=$(OutDir);LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
+    <OutputPath>$(OutDir)\VsixV3</OutputPath>
```

Comment:
The name `Insertion` will be ambiguous since we have Nuget insertion packages to insert into classic installer and since we have normal Vsixes the name `VSIX` will again be ambiguous. I was thinking of `ModernVsix` but again if there is a new version of VSIX then will be ambiguity!! Hence I went with a name which tells its exact format and purpose.